deploy/eck-stack/charts/eck-elasticsearch/templates/ingress.yaml

```diff
@@ -0,0 +1,46 @@
+{{- if .Values.ingress.enabled -}}
+{{- $pathType := .Values.ingress.pathType | default "Prefix" -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "elasticsearch.fullname" . }}
+  labels:
+    {{- with .Values.ingress.labels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  annotations:
+    eck.k8s.elastic.co/license: basic
```

Comment:
why is the annotation being set as basic always ?

Reply:
Good point, this isn't required, and I removed it, but it also made me think about the fact that we should be added the "default" helm labels as well, which I've done, and added tests for both es/kb ingress.